src/providers.ts

```diff
@@ -152,6 +152,8 @@ export class SimpleLanguageInfoProvider {
   }
 }
 
+const TOKENIZE_TIMEOUT_MS = 500;
```

Comment:
Could you please add a comment to explain how this is used?

Reply:
Good idea: included a link.